documentation/src/main/wiki/Handlebars.md

```diff
@@ -0,0 +1,21 @@
+# Handlebars
```

Comment:
Please add this file to `_Sidebar.md` in order to display it among other positions in Wiki page.